2024-10-08 15:23:26,448 INFO    MainThread:22368 [wandb_setup.py:_flush():76] Current SDK version is 0.16.5
2024-10-08 15:23:26,448 INFO    MainThread:22368 [wandb_setup.py:_flush():76] Configure stats pid to 22368
2024-10-08 15:23:26,448 INFO    MainThread:22368 [wandb_setup.py:_flush():76] Loading settings from C:\Users\valer\.config\wandb\settings
2024-10-08 15:23:26,448 INFO    MainThread:22368 [wandb_setup.py:_flush():76] Loading settings from c:\Users\valer\Desktop\macc-project\Model\wandb\settings
2024-10-08 15:23:26,448 INFO    MainThread:22368 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2024-10-08 15:23:26,448 INFO    MainThread:22368 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2024-10-08 15:23:26,449 INFO    MainThread:22368 [wandb_setup.py:_flush():76] Applying login settings: {'api_key': '***REDACTED***'}
2024-10-08 15:23:26,449 INFO    MainThread:22368 [wandb_setup.py:_flush():76] Applying login settings: {'api_key': '***REDACTED***'}
2024-10-08 15:23:26,449 INFO    MainThread:22368 [wandb_setup.py:_flush():76] Applying login settings: {}
2024-10-08 15:23:26,449 INFO    MainThread:22368 [wandb_init.py:_log_setup():527] Logging user logs to .\wandb\run-20241008_152326-dwrt9is4\logs\debug.log
2024-10-08 15:23:26,450 INFO    MainThread:22368 [wandb_init.py:_log_setup():528] Logging internal logs to .\wandb\run-20241008_152326-dwrt9is4\logs\debug-internal.log
2024-10-08 15:23:26,450 INFO    MainThread:22368 [wandb_init.py:_jupyter_setup():473] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x0000026ECEB62CB0>
2024-10-08 15:23:26,451 INFO    MainThread:22368 [wandb_init.py:init():567] calling init triggers
2024-10-08 15:23:26,451 INFO    MainThread:22368 [wandb_init.py:init():574] wandb.init called with sweep_config: {}
config: {}
2024-10-08 15:23:26,451 INFO    MainThread:22368 [wandb_init.py:init():617] starting backend
2024-10-08 15:23:26,451 INFO    MainThread:22368 [wandb_init.py:init():621] setting up manager
2024-10-08 15:23:26,454 INFO    MainThread:22368 [backend.py:_multiprocessing_setup():105] multiprocessing start_methods=spawn, using: spawn
2024-10-08 15:23:26,473 INFO    MainThread:22368 [wandb_init.py:init():629] backend started and connected
2024-10-08 15:23:26,478 INFO    MainThread:22368 [wandb_run.py:_label_probe_notebook():1299] probe notebook
2024-10-08 15:23:26,479 INFO    MainThread:22368 [wandb_run.py:_label_probe_notebook():1309] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2024-10-08 15:23:26,480 INFO    MainThread:22368 [wandb_init.py:init():721] updated telemetry
2024-10-08 15:23:26,529 INFO    MainThread:22368 [wandb_init.py:init():754] communicating run to backend with 90.0 second timeout
2024-10-08 15:23:27,187 INFO    MainThread:22368 [wandb_run.py:_on_init():2344] communicating current version
2024-10-08 15:23:27,353 INFO    MainThread:22368 [wandb_run.py:_on_init():2353] got version response upgrade_message: "wandb version 0.18.3 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2024-10-08 15:23:27,353 INFO    MainThread:22368 [wandb_init.py:init():805] starting run threads in backend
2024-10-08 15:23:28,929 INFO    MainThread:22368 [wandb_run.py:_console_start():2323] atexit reg
2024-10-08 15:23:28,929 INFO    MainThread:22368 [wandb_run.py:_redirect():2178] redirect: wrap_raw
2024-10-08 15:23:28,929 INFO    MainThread:22368 [wandb_run.py:_redirect():2243] Wrapping output streams.
2024-10-08 15:23:28,929 INFO    MainThread:22368 [wandb_run.py:_redirect():2268] Redirects installed.
2024-10-08 15:23:28,930 INFO    MainThread:22368 [wandb_init.py:init():848] run started, returning control to user process
2024-10-08 15:23:28,930 INFO    MainThread:22368 [wandb_run.py:_finish():2052] finishing run valeriodstfn/fnn/dwrt9is4
2024-10-08 15:23:28,931 ERROR   MainThread:22368 [jupyter.py:save_history():437] Run pip install nbformat to save notebook history
2024-10-08 15:23:28,931 INFO    MainThread:22368 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2024-10-08 15:23:28,931 INFO    MainThread:22368 [wandb_init.py:_jupyter_teardown():455] cleaning up jupyter logic
2024-10-08 15:23:28,931 INFO    MainThread:22368 [wandb_run.py:_atexit_cleanup():2292] got exitcode: 0
2024-10-08 15:23:28,931 INFO    MainThread:22368 [wandb_run.py:_restore():2275] restore
2024-10-08 15:23:28,931 INFO    MainThread:22368 [wandb_run.py:_restore():2281] restore done
2024-10-08 15:23:32,734 INFO    MainThread:22368 [wandb_run.py:_footer_sync_info():3879] logging synced files
